pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/jstl/JstlTypeConverter.java

```diff
@@ -107,6 +115,12 @@ protected Integer coerceToInteger(Object value) {
     if (value instanceof Time) {
       return (int) (((Time) value).toLocalTime().toNanoOfDay() / 1_000_000);
     }
+    if (value instanceof Date) {
```

Comment:
Or should I use `Date.class.equals(value.getClass()))` to avoid matching a Timestamp?